bundle-workflow/assemble.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Copyright OpenSearch Contributors.
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+DIR="$(dirname "$0")"
+python3 "$DIR/python/assemble.py" $@
```

Comment:
I'd like us to rename this `python` directory because it's a property of the language, not a thing. Let's just call it `lib`?

Reply:
updated this.  We already have a folder in there named lib, so I renamed that to src.